cmd/imagedigestexporter/main.go

```diff
@@ -65,5 +65,18 @@ func main() {
 	if err != nil {
 		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
 	}
-	fmt.Println(string(imagesJSON))
+	log.Printf("Image digest exporter output: %s ", string(imagesJSON))
+	f, err := os.OpenFile("/builder/termination-log", os.O_WRONLY|os.O_CREATE, 0666)
+	if err != nil {
+		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
+	}
+	defer f.Close()
+
+	_, err = f.Write(imagesJSON)
+	if err != nil {
+		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
+	}
+	if err := f.Sync(); err != nil {
+		log.Fatalf("Unexpected error converting images to json %v: %v", output, err)
+	}
```

Comment:
maybe in another iteration/PR it would be great to see some tests for this - esp as it gets a bit more complex (even if its only a bit!)